aws-ecsfargate-terraform/README.md

```diff
@@ -242,3 +242,27 @@ If you want to view the logs for your SCIM bridge within AWS, go to **Cloudwatch
 If you open your SCIM bridge domain in a browser and see a `Sign In With 1Password` button, the `scimsession` file was not properly installed. Due to the nature of the ECS deployment, **this “sign in” option cannot be used** to complete the setup of your SCIM bridge.
 
 To fix this, [copy the `scimsession` file](#11-copy-scimsession-file) again and stop the ECS task for your SCIM bridge. The ECS service will ensure that the task is restarted to reboot your SCIM bridge and apply the changes.
+
+#### If unable to access AWS Secrets Manager
+
+The `aws_security_group.service` resource creates a security group that restricts outbound traffic from the Fargate task such that 1Password SCIM Bridge may only connect to the 1Password service over HTTPS on this port. If the VPC in which the Fargate task is deployed has not been configured to support DNS resolution through the Amazon DNS server, this security group will also restrict DNS resolution.
```

Comment:
To simplify the language a bit:

```suggestion
Because the `aws_security_group.service` resource creates a security group that restricts outbound traffic from the Fargate task, 1Password SCIM Bridge may only connect to the 1Password service over HTTPS on this port. If the VPC where the Fargate task is deployed hasn't been configured to support DNS resolution through the Amazon DNS server, this security group will also restrict DNS resolution.
```